src/de.rs

```diff
@@ -96,16 +100,16 @@ pub trait Decoder: Sized {
     ) -> Result<types::ObjectIdentifier, Self::Error>;
     /// Decode a `SEQUENCE` identified by `tag` from the available input. Returning
     /// a new `Decoder` containing the sequence's contents to be decoded.
-    fn decode_sequence<D, DF, F>(
+    fn decode_sequence<const RC: usize, const EC: usize, D, DF, F>(
```

Comment:
Please add either documentation that explains each generic, or use full length names. There's too many generics for it to not be either. If you use full length names, you don't have to use the full names in every `impl`, just in the definition here.